roles/aces-marketplace/templates/backend.service.j2

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Aces Marketplace Backend
+
+[Service]
+Restart=always
+WorkingDirectory=/app/aces-marketplace/backend
```

Comment:
this working directory is incorrect, needs to be `/opt/{{ app_service_name }}/backend`